assignAgenttoTicketPart2.js

```diff
@@ -0,0 +1,57 @@
+import assignAgenttoTicket from './assignAgenttoTicket.js';
+import getAgents from './getAgents.js';
+import getTicketsByStatus from './getTicketsByStatus.js';
+import makeZendeskRequest from './makeZendeskRequest.js';
+
+const assignAgenttoTicketPart2 = async (agentLastName) => {
+
+  if (agentLastName == undefined) {
+    agentLastName = '';
+  }
```

Comment:
can't we just compare if agentLastName is undefined later on? this seems like an redundant step